util/src/commonMain/kotlin/care/data4life/sdk/log/Log.kt

```diff
@@ -17,9 +17,11 @@
 package care.data4life.sdk.log
 
 import kotlin.jvm.JvmStatic
+import kotlin.native.concurrent.ThreadLocal
 
 class Log {
 
+    @ThreadLocal
```

Comment:
To answer the question from the PR before - this was done to get rid of the "Variable in singleton without @ThreadLocal can't be changed after initialization" compiler warning. Technically not necessary but can be confusing a lot while dealing with Errors in iOS and Coroutines.

Reply:
👍